tests/conftest.py

```diff
@@ -10,3 +15,68 @@ def pytest_addoption(parser):
         type=pathlib.Path,
         help="Root of the ISO file tree."
     )
+
+# Fixtures {{{
+
+@pytest.fixture
```

Comment:
I know you just moved the code, but this one can have a `scope="module"` as well I believe :)

Reply:
Done